lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/domain/UKAddresses.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.di.ipv.cri.kbv.api.domain;
+
+public class UKAddresses {
```

Comment:
This should ideally be using the CanonicalAddress from the Address CRI - we should be storing UPRN where available

Reply:
Also CountryCode in ISO 3-Character format (in this case, can be defaulted to GBR)

Reply:
As discussed with @idrop and the wider team, the address canonical form is not required at this point for the KBV CRI. It will be handled as part of a new ticket on low priority - https://govukverify.atlassian.net/browse/KBV-289